modules/web/src/app/core/services/auth/service.ts

```diff
@@ -190,18 +190,15 @@ export class Auth {
   }
 
   private _deleteToken(): void {
-    if (this._cookieService.check(this._cookie.token)) {
-      this._cookieService.delete(this._cookie.token);
-      return;
-    }
+    this._cookieService.delete(this._cookie.token, '/');
+    this._tokenService.token = null;
```

Comment:
I see Setter accepts string as an argument and it works because of not having ""strictNullChecks": false" by default should we update signature of setter to accept null too for safety reasons?

File: TokenService

```ts
set token(token: string | null) {
  ...
}
```